### PR TITLE
Release 4.1.3

### PR DESCRIPTION
**Pull request by ashcrow**
At step 4 of [release](https://github.com/coreos/afterburn/blob/master/docs/release-guide.md#4-get-the-pr-reviewed-approved-and-merged).

# Changes
- cargo: update dependencies (slog, serde, serde_derive, nix, mime)

### CODE REVIEW

**Review by bgilbert — APPROVED** [review]
LGTM

**Comment by lucab** [comment]
I usually assemble a short list of changes, like in https://github.com/coreos/afterburn/pull/257.
I think this one only contains dependencies updates, thus just mentioning them as a single entry would be enough: https://github.com/coreos/afterburn/compare/v4.1.2...master

File: Cargo.toml [hunk 1]
```diff
@@ -41,22 +51,12 @@ tempfile = "^3.0.9"
...
 version = "2.5"
 features = ["max_level_trace", "release_max_level_info"]
 
+[features]
+cl-legacy = ["update-ssh-keys"]
+
 [dev-dependencies]
 mockito = "^0.17.1"
-
-[package.metadata.release]
```
Comment (lucab) [inline Cargo.toml:54]: Did you move these sections by hand or did something re-organized them?

Comment (ashcrow) [reply Cargo.toml:54]: `cargo release` reorganized them

Comment (lucab) [reply Cargo.toml:54]: I think it's an unexpected side-effect, I'll check with upstream later.

I checked that there was no functional change except the different order, so we can proceed anyway.

**Review by lucab — APPROVED** [review]
LGTM! @ashcrow do you have permissions to self-merge this?

**Comment by ashcrow** [comment]
@lucab I have the power!

**Comment by lucab** [comment]
@ashcrow feel free to self-merge and continue with the release, at your own pace.

**Pull request merged**